src/com/facebook/buck/testrunner/TestNGRunner.java

```diff
@@ -92,7 +92,6 @@ public void run() throws Throwable {
   private XmlSuite createXmlSuite(Class<?> c) {
     XmlSuite xmlSuite = new XmlSuite();
     xmlSuite.setName("TmpSuite");
-    xmlSuite.setTimeOut(String.valueOf(defaultTestTimeoutMillis));
```

Comment:
why this change is part of this PR? Is that necessary?

Reply:
It's a separate change, and I can make a separate 1-line PR instead? We originally had these together in 1 commit, which is why they're together.

Reply:
Or, could I combine this timeout removal with some of the "structural refactors" from #1663 ?

Reply:
Yes, please, move it to a separate change. In case it causes any problems we can revert this change only without touching logic around skipped tests